### PR TITLE
Bug 1879902, added metrics for 3.11 route maximums + Fixed footnotes

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1879902

@mohit-sheth @chaitanyaenr PTAL, specifically at the new Route Maximums section 

Preview build: http://file.rdu.redhat.com/~ahardin/Sept-29-2020/3-11-route-limits/scaling_performance/cluster_maximums.html#scaling-performance-3-11-route-maximums

### CODE REVIEW

File: scaling_performance/cluster_maximums.adoc [hunk 1]
```diff
@@ -146,14 +149,41 @@ application requirements.]
...
+iptables. The number of back-ends of a given service impact the size of the
+endpoints objects, which impacts the size of data that is being sent all over
+the system.
+--
+[[scaling-performance-3-11-route-maximums]]
+=== Route Maximums
+
+In 3.11.53, router tests were completed in a 3-node environment on Amazon Web
+Services (AWS). There were 100 HTTP routes, specifically 100 back-end Nginx pods,
+with `keepalive` set to `100`, and 200 parallel connections per route. The
```
Comment (mohit-sheth) [inline scaling_performance/cluster_maximums.adoc:180]: you can remove 200 parallel connections per route as we are stating numbers for 1,40 and 200 connections per target below

File: scaling_performance/cluster_maximums.adoc [hunk 2]
```diff
@@ -146,14 +149,40 @@ application requirements.]
...
+application requirements.
+3. Each Service port and each Service back-end has a corresponding entry in
+iptables. The number of back-ends of a given service impact the size of the
+endpoints objects, which impacts the size of data that is being sent all over
+the system.
+--
+[[scaling-performance-3-11-route-maximums]]
+=== Route Maximums
+
+In 3.11.53, router tests were completed in a 3-node environment on Amazon Web
```
Comment (adellape) [inline scaling_performance/cluster_maximums.adoc:178]: Should this be `{product-title} 3.11.53`?

**Review by adellape — APPROVED** [review]
:+1:

File: scaling_performance/cluster_maximums.adoc (hunk 2 above)
Comment (ahardin-rh) [reply scaling_performance/cluster_maximums.adoc:178]: oops! yes! Updating . . .

**Review by chaitanyaenr — APPROVED** [review]
LGTM.

**Comment by ahardin-rh** [comment]
@mffiedler PTAL Thank you!

**Review by mffiedler — APPROVED** [review]
/LGTM

**Pull request merged**